### PR TITLE
Add & Remove todo_list PR

### PR DESCRIPTION
**Pull request by Kossi-stack**
**This Project following requirements are :**

 -   Implemented a function for the "Clear all completed" button.

 -   New tasks have the property completed set to false and the property index set to the value of the new array length.
 
 -  Deleting a task update all remaining items' indexes, so they represent the current list order and are unique.

 -  All changes to the To-Do List are saved in local storage.

 -  Removed all hardcoded items from the tasks array.

 -  Created a new JavaScript file for the new functionality.

 -  Implemented a function for adding a new task (add a new element to the array).

 -  Implemented a function for editing task descriptions.

 -  Implemented a function for deleting a task (remove an element from the array).

### CODE REVIEW

**Review by carlos-ssh — APPROVED** [review]
Good work!

File: src/deleteOne.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import showTasks from './showTasks.js';
+import { setLocalStorage } from './setLocalStorage.js';
+import getTasksFromLocalStorage from './getTasks.js';
+// eslint-disable-next-line import/no-cycle
```
Comment (carlos-ssh) [inline src/deleteOne.js:4]: **You are missing to enable your ESLint into your pull Request.**
- Enable your Linter after that fix issues if it's needed.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Kossi-stack , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
### Hi @Kossi-stack  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**